src/SFA.DAS.EAS.Portal.Client/Application/Queries/IGetAccountQuery.cs

```diff
@@ -0,0 +1,11 @@
+using System.Threading;
+using System.Threading.Tasks;
+using SFA.DAS.EAS.Portal.Client.Types;
+
+namespace SFA.DAS.EAS.Portal.Client.Application.Queries
+{
+    internal interface IGetAccountQuery
```

Comment:
Can this be named IAccountQuery ? The Get is a method of the interface so doesn't need to be in the interface name.

Reply:
Currently, we only have one query, but I envisage that each query will be contained within its own object, rather than having multiple query methods within a type ([CQRS, rather than CQS](http://cqrs.nu/Faq)). In which case, I think it's more appropriate to name the class and interface after the query, rather than the entity/DTO, as we could have multiple queries for a single entity/DTO, in which case there would be naming collisions,

Reply:
We don't have commands since we don't have a write model so CQRS doesn't apply.  My comment was about the naming of the interface.  Do we even need this interface?  The concrete is just a helper method for simplifying the query against the repository.  In that case its a query service so a better name would IAccountQueries or IAccountQueryService with an initial method GetActiveAccount() [or GetAccount()  with it implied that the account is active]  since that is whet this helper object is doing.  Then we may add methods like GetDeletedAccount() or FindByName()  methods etc...

Reply:
Yes, the client really has no need for queries or commands. It's likely that it'll only have to fetch an account, so we might as well keep it simple and just have a method on the repository. That allows us to remove some infrastructure that wasn't adding any value.

Reply:
I don't mind either way on the naming of this one.

Reply:
A `Get` method has been added to the accounts repository, and the query removed.